src/styles/_variables.scss

```diff
@@ -38,7 +38,7 @@ $field-disabled-opacity: 0.7;
 $font-size-xx-large: 24px;
 $font-size-x-large: 18px;
 $font-size-large: 16px;
-$font-size-medium: 13px;
+$font-size-medium: 14px;
```

Comment:
seems like you're changing a font-size that can be used on the app, according to the mockup:
https://projects.invisionapp.com/d/main#/console/14289248/297316534/preview

@fmontes please confirm this!